### DISCUSSION TITLE
[:0100000b] lua call [0 to :100000b : 0 msgsz = 56] error

### DISCUSSION THREAD

**Discussion by liyqxtu** [cloudwu/skynet#1739] [General]
[:0100000b] LAUNCH snlua console
[:0100000b] lua call [0 to :100000b : 0 msgsz = 56] error : ./lualib/skynet.lua:860: ./lualib/skynet.lua:331: ./lualib/skynet\socket.lua:365: assertion failed!
stack traceback:
        [C]: in function 'assert'
        ./lualib/skynet\socket.lua:365: in function 'skynet.socket.readline'
        ./service/console.lua:16: in upvalue 'f'
        ./lualib/skynet.lua:254: in function <./lualib/skynet.lua:253>
stack traceback:
        [C]: in function 'assert'
        ./lualib/skynet.lua:860: in function 'skynet.dispatch_message'
[:0100000c] LAUNCH snlua debug_console 8000

**Comment by cloudwu** [comment]
此类问题不要提 issue , 用 dissection 。

**Reply by liyqxtu** [reply]
大佬，这个是没有安装readline导致的么？
